internal/hcsoci/create.go

```diff
@@ -109,7 +109,7 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	if coi.HostingSystem != nil {
 		n := coi.HostingSystem.ContainerCounter()
 		if coi.Spec.Linux != nil {
-			resources.containerRootInUVM = "/run/gcs/c/" + strconv.FormatUint(n, 16)
+			resources.containerRootInUVM = "/run/gcs/c/" + createOptions.ID
```

Comment:
Lets define these as

```go
var (
    lcowRootInUVM = "/run/gcs/c/%s"
    wcowRootInUVM = "C:\c\%d"
)
```

And then change the calls below to be `fmt.Sprintf(<const>, id)`